eparticlectr/eparticlectr.cpp

```diff
@@ -357,76 +255,27 @@ void eparticlectr::finalizeextr( uint64_t proposal_id ) {
                 });
             }
         }
-        // Reduce staking time for vote winners to 5 days
-        else if (approved && !vote_it->is_editor) {
+        // Reduce staking time for vote winners to 5 days. Keep losers the same
+        else if (!vote_it->is_editor) {
```

Comment:
I wonder if we should keep here the approved or not. the if still has the approved condition, not sure if there can be some race condition where this approved can guard

Reply:
I will fix the logic

Reply:
actually, the `if (vote_it->approve == approved)` logic below handles it